lib/digest.js

```diff
@@ -0,0 +1,63 @@
+/*!
+ * Copyright (c) 2022 Digital Bazaar, Inc. All rights reserved.
+ */
+import jsonld from 'jsonld';
+import {encode as base58Encode} from 'base58-universal';
+import {sha256digest} from './sha256digest.js';
+
+const URDCA_2015_ALG = 'urdca2015';
+// Multicodec table entry code 0xb403, converted to varint
+const URDCA2015_HEADER = new Uint8Array([131, 232, 2]);
+
+/**
+ * Performs a combined canonicalize + multihash operation.
+ *
+ * @param {object} options - Options hashmap.
+ * @param {object} options.input - JSON-LD object.
+ * @param {string} [options.canonizeAlgorithm] - Canonicalization algorithm.
+ * @param {Function} options.documentLoader - JSON-LD document loader.
+ *
+ * @returns {Promise<string>} Returns the canonhash digest, base58-encoded.
```

Comment:
`canonhash` seems weird? Is that intentional?

```suggestion
 * @returns {Promise<string>} Returns the canonical hash digest, base58-encoded.
```